content/applications/finance/fiscal_localizations/india.rst

```diff
@@ -166,6 +166,45 @@ Cancel`.
 
 .. _india/verify-e-invoice:
 
+Managing Negative Lines in E-Invoice
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The government portal prohibits the submission of data with negative line values.
+To facilitate the inclusion of discount lines, effective handling of negative entries becomes essential.
+
+Negative Lines Conversion in E-Invoice
+**************************************
+
+When dealing with E-invoice entries, there may be scenarios where negative lines are encountered. These negative lines are typically employed to represent discounts or adjustments associated with specific products HSN and tax rate or global discounts.
+
+Consider the following example:
```

Comment:
I would put the example in a an "example" box

Reply:
done